src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1137,7 +1143,7 @@ def __del__(self):
         request.commandid = self.path[-1][1]
         try:
             self.service.delete_command_arguments(request)
-        except ValueError:
+        except BaseException:
             # "Cannot invoke RPC on closed channel!"
             pass
```

Comment:
```suggestion
        except BaseException as exc:
            logger.info("__del__ %s: %s" % (type(exc).__name__, exc))
```
Removing the commented line as it is no longer applicable, and adding a logging report of the BaseException that was caught (particularly as this is a very general exception catch).